code/game/objects/items/rcd/RCD.dm

```diff
@@ -215,6 +216,7 @@
 		return FALSE
 	rcd_results["[RCD_DESIGN_MODE]"] = mode
 	rcd_results["[RCD_DESIGN_PATH]"] = rcd_design_path
+	rcd_results["[RCD_BUILD_DIRECTION]"] = selected_direction || user.dir
```

Comment:
I'd make you change the "[]" but it's used everywhere in this file so dealer's choice

Reply:
its done